integration-tests/integration/statements/request-removal.spec.js

```diff
@@ -17,14 +18,21 @@ context('Request removal', () => {
         incidentDate: moment('2019-01-22 09:57:40.000'),
         agencyId: 'MDI',
         involvedStaff: [
-          { name: 'Emily Jones', email: 'Emily@gov.uk', staffId: 5, username: 'EMILY_JONES', verified: true },
+          {
+            name: 'Emily Jones',
+            email: 'Emily@gov.uk',
+            staffId: 5,
+            username: 'EMILY_JONES',
+            verified: true,
+          },
         ],
       })
       .then(result => result.EMILY_JONES)
 
   it('A user can submit their statement removal request', () => {
     seedReport().then(statementId => {
-      const requestRemovalPage = RequestRemovalPage.visit(statementId)
+      cy.task('updateStatementId', { oldId: statementId, newId: 10 })
+      const requestRemovalPage = RequestRemovalPage.visit(10, 'mrsSjv9LqjClfvuNubYx4ACALTltJvqLmXYE0TKz6Xo=')
```

Comment:
Wanted to reuse hash functions but can't as browser crypto and server crypto have different impls.
This is just the hardcoded hash for statement 10 with the local dev hardcoded secret

Reply:
Could you ask the server for the hash value instead? 
Looks like cy.task() can return a value: https://docs.cypress.io/api/commands/task#Yields

> The task plugin event handler can return a value or a promise. The command will fail if undefined is returned or if the promise is resolved with undefined. This helps catch typos or cases where the task event is not handled.

Reply:
excellent suggestion!